src/App.js

```diff
@@ -1,47 +1,77 @@
 import React from 'react';
 import './App.scss';
+import PropTypes from 'prop-types';
 
-const misha = {
+export const misha = {
```

Comment:
```suggestion
 const misha = {
```
You don't need to export this. Pass these objects to the `Person` component. Read the task description more attentively.

You should create the component `Person` that will receive the object with content from the outside (from the `App` component) and render some content inside. Then you should insert three `Person` components and pass there three different objects